lang/languages.json

```diff
@@ -1,59 +1,59 @@
 {
-    "af": "Afrikaans",
-    "ar": "العربية",
-    "be": "Беларуская",
-    "bg": "Български",
-    "ca": "Català",
-    "cs": "Čeština",
-    "da": "Dansk",
-    "de": "Deutsch",
-    "dsb": "Dolnoserbšćina",
-    "el": "Ελληνικά",
-    "en": "English",
-    "enGB": "English (United Kingdom)",
-    "eo": "Esperanto",
-    "es": "Español",
-    "esUS": "Español (Latinoamérica)",
-    "et": "Eesti",
-    "eu": "Euskara",
-    "fa": "فارسی",
-    "fi": "Suomi",
-    "fr": "Français",
-    "frCA": "Français (Canada)",
-    "gl": "Galego",
-    "he": "עברית",
-    "hi": "हिन्दी",
-    "hr": "Hrvatski",
-    "hsb": "Hornjoserbšćina",
-    "hu": "Magyar",
-    "hy": "Հայերեն",
-    "is": "Íslenska",
-    "it": "Italiano",
-    "ja": "日本語",
-    "kab": "Taqbaylit",
-    "ko": "한국어",
-    "lt": "Lietuvių",
-    "lv": "Latviešu",
-    "ml": "മലയാളം",
-    "mn": "Монгол",
-    "mr": "मराठी",
-    "nl": "Nederlands",
-    "oc": "Occitan",
-    "pl": "Polski",
-    "pt": "Português",
-    "ptBR": "Português (Brasil)",
-    "ro": "Română",
-    "ru": "Русский",
-    "sc": "Sardu",
-    "sk": "Slovenčina",
-    "sl": "Slovenščina",
-    "sq": "Shqip",
-    "sr": "Српски",
-    "sv": "Svenska",
-    "te": "తెలుగు",
-    "tr": "Türkçe",
-    "uk": "Українська",
-    "vi": "Tiếng Việt",
-    "zhCN": "中文（简体）",
-    "zhTW": "中文（繁體）"
+  "af": "Afrikaans",
```

Comment:
Please don't reindent the whole file. Just add the new entry.

Reply:
Sorry, my bad.